### DISCUSSION TITLE
Provide an option not to animate creatures on well screen

### DISCUSSION THREAD

**Discussion by jez9999** [ihhub/fheroes2#8199] [Ideas]
### Preliminary checks

- [X] I've checked that there aren't [**other open issues**](https://github.com/ihhub/fheroes2/issues?q=is%3Aissue) on the same topic.

### Describe the problem requiring a solution

The original game didn't animate all the creatures on the well screen, which I find very distracting.

### Describe the possible solution

Provide an option to turn off these creature animations on the well screen.

**Comment by ihhub** [comment]
Hi @jez9999 , except being used to it whats wrong with the animation?

**Comment by jez9999** [comment]
It's really distracting, I'm trying to focus on reading the monster info.

**Comment by ihhub** [comment]
Not sure how it can be distracting if monsters do **not** cover their stats by their animation. Also, the well now allows to buy monsters directly from there. All other players love this change.

**Comment by jez9999** [comment]
Ehh, guess this is the sort of reason I don't like fheroes2.  Look, you guys should've made a faithful reproduction of the original and then added stuff to it, providing an option to turn each thing off.  People who wanted a faithful repro (AI excepted) could've had it then.

**Comment by ihhub** [comment]
@jez9999 , we aren't going to reproduce some wacky UI or make a game with 100s of options. This particular animation affects nothing but esthetical part of the game. Like I mentioned earlier: please do not look at fheroes2 and criticize as before you used to see or play it. As of now no proper argument was given why we should do this.

**Comment by jez9999** [comment]
Ludicrous.  It would be easy for you to have a config file with a flag and just disable the animations if it were turned off.  You don't need UI with 100s of options.  As for not providing a bunch of config options, frankly, why not?

**Comment by Mr-Bajs** [comment]
> Ludicrous. It would be easy for you to have a config file with a flag and just disable the animations if it were turned off. You don't need UI with 100s of options. As for not providing a bunch of config options, frankly, why not?

It would be easy for you to add this feature then? Why not?

**Comment by jez9999** [comment]
You'd accept the pull request?

**Comment by ihhub** [comment]
@jez9999 ,

First of all, so far we haven't received any objective explanation about why we need this option except your subjective opinion. "I don't like it" is not an argument why we need to spend development time doing this. Secondly, [here](https://github.com/ihhub/fheroes2/wiki/Adding-a-new-option-into-the-game) is a guide regarding any new option being added in the engine. Make sure that you fulfil **all** the requirements before you even open a pull request. Thirdly, we accept pull requests but they must meet quality criteria for the code.

Therefore, based on the conversation I am converting this issue into a discussion.
